student/src/test/java/tds/student/services/PrintServiceTest.java

```diff
@@ -41,7 +41,7 @@
   private static final Logger _logger               = LoggerFactory.getLogger (PrintServiceTest.class);
 
   @Autowired
-  PrintService          printService ;
+  PrintServiceImpl printService ;
```

Comment:
Nice catch, IntelliJ helpfully refactored this property datatype (as well as some of the other references that I actually caught during end-to-end testing)